server/routes/bookAVisit.ts

```diff
@@ -262,7 +270,7 @@ export default function routes(
         visitSessionData.visitStatus = visitStatus
       }
 
-      auditService.reservedVisit(
+      await auditService.reservedVisit(
```

Comment:
I've added `await` wherever `auditService` methods are called as it it looked like they should be there to be consistent with how our other services are called?